tests/integration/utils.sh

```diff
@@ -31,13 +31,41 @@ function build_distro_image()
 {
   local distro="$1"
   local file="${CONTAINER_DIR}/Containerfile_${distro}"
+  local id
+  local size
+  local size_unit
 
   image_build --file "$file" --tag "kw-${distro}"
 
   if [[ "$?" -ne 0 ]]; then
     fail "(${LINENO}): Error building the image for distribution ${distro}"
     return "$?"
   fi
+
+  size_unit=$(podman image inspect --format '{{ .Size }}' "kw-${distro}")
+  id=$(podman image inspect --format '{{ .Id }}' "kw-${distro}")
+
+  # Converting size in bytes to the appropriate units
+  if [ "${size_unit}" -lt 1024 ]; then
```

Comment:
Also, could you use `[[]]` instead of `[]` in the if statements?

It will look like this:

```
  if [[ "$size_unit" -lt 1024 ]]; then
    size="${size_unit} B"
  elif [[ "$size_unit" -lt "$(bc <<< 1024^2)" ]]; then
    size_unit=$(printf "scale=2; %s / %s\n" "$size_unit" "1024" | bc)
    size="${size_unit} KB"
```